src/node/util.ts

```diff
@@ -509,3 +509,17 @@ export const isFile = async (path: string): Promise<boolean> => {
     return false
   }
 }
+
+/**
+ * Escapes any HTML string special characters, like &, <, >, ", and '.
+ *
+ * Source: https://stackoverflow.com/a/6234804/3015595
+ **/
+export function escapeHtml(unsafe: string): string {
+  return unsafe
+    .replace(/&/g, "&amp;")
+    .replace(/</g, "&lt;")
+    .replace(/>/g, "&gt;")
+    .replace(/"/g, "&quot;")
+    .replace(/'/g, "&#039;")
```

Comment:
small nit:
I think we could use `&apos;` here instead of `&#039;`, the former is more immediately recognizable in text!

The reason `&#039` is sometimes recommended is because of old browsers that don't support `&apos`, but we basically target the latest versions of Chrome/FF/Safari and have no need to support IE8, so *should* be fine!

Reply:
TIL! Thanks for the suggestion. I'll change that then merge!